src/main/webapp/header.jsp

```diff
@@ -190,6 +191,47 @@ finally{
       <link type="text/css" href="<%=urlLoc %>/css/imageEnhancer.css" rel="stylesheet" />
 
       <script src="<%=urlLoc %>/javascript/lazysizes.min.js"></script>
+      <%
+        if(user != null && !loggingOut){
+      %>
+        <script type="text/javascript">
+        $(document).ready(function() {
+
+            // Session warning times in minutes
+            var warningTime = <%=sessionWarningTime %>;
+
+            function showWarning() {
+              $('#sessionModal').modal('show');
+            }
+
+            function extendSession() {
+                $.get("ExtendSession", function() {
```

Comment:
i am thinking this url should be _absolute_ (e.g. `/ExtendSession`), as there are pages (like `/encounters/encounter.jsp`) which will get a 404 if this is relative.

Reply:
i have updated extendSession to use similar approach which is already being used in project kindly let me know if its good

Reply:
ah right, that global js variable seems like the better way to do it. thanks.